package.json

```diff
@@ -107,7 +107,7 @@
 		"jsdoc": "=3.4.3",
 		"jsdox": "=0.4.10",
 		"lint-staged": "=6.1.0",
-		"lisk-js": "1.0.0-beta.2",
+		"lisk-elements": "1.0.0-beta.4",
```

Comment:
shouldn't there be a '=' before version?

Reply:
Actually there shouldn't be an `=` before any of them. No prefix means "exact" and `=` doesn't isn't defined in the npm documentation.

Reply:
See https://docs.npmjs.com/files/package.json#dependencies